test/entity/multipleFieldsEntity.js

```diff
@@ -1,135 +1,163 @@
 const { entity } = require('../../src/entity')
 const { field } = require('../../src/field')
 const assert = require('assert')
+const { id } = require('../../src/id')
 
 describe('An entity', () => {
+  describe('with multiple fields', () => {
+    const NewEntity = entity('New Entity', {
+      f1: field(String),
+    })
+
+    const givenAnEntityWithMultipleFields = () => {
+      const AnEntity = entity('An entity', {
+        field1: field(Number),
+        field2: field(String),
+        field3: field(Date),
+        field4: field(Boolean),
+        field5: field(NewEntity),
+        field6: field([NewEntity]),
+        field7: id(Number),
+        field8: id(String),
+        field9: field(Number, { isId: true }),
+        field10: id(Date),
+        field11: field(Boolean, { isId: true }),
+      })
+      return new AnEntity()
+    }
+
+    it('should initiate', () => {
+      //given
+      const instance = givenAnEntityWithMultipleFields()
+      //then
+      assert.equal(instance.meta.name, 'An entity')
+    })
+
+    it('should set a value to multiple fields', () => {
```

Comment:
ID fields have been added to the entity to be tested, but the values and behaviors are not being tested in all scenarios (eg `field7` to `field11`)

Reply:
I changed the tests, can you give me a new feedback on those?
thanks @dalssoft